app/listeners/notification_listener.rb

```diff
@@ -1,4 +1,18 @@
 class NotificationListener < BaseListener
+  def conversation_bot_handoff(event)
```

Comment:
@Jatin-MYST Sorry for the late response. It seems like a good enhancement. Could you please add the specs for it? We will test and merge it after that.

Reply:
@muhsin-k 
Hey, apologies but I am unable to add the specs.
I am not that familiar with RoR and the testing package in use.
I have been using this change on my self-hosted instance ever since i submitted this PR and it has been working fine.
Can someone please add the tests instead?